regression-test/data/tpcds_sf0.1_p1/sql/q01.out

```diff
@@ -0,0 +1,3 @@
+-- This file is automatically generated. You should know what you did if you want to edit this
+-- !q01 --
+
```

Comment:
Is it empty? If so, I think this scale doesn't work for correctness validation

Reply:
> Is it empty? If so, I think this scale doesn't work for correctness validation

It's so strange. I'll go check if some tables of the generated tpcds data are empty.

Reply:
> > Is it empty? If so, I think this scale doesn't work for correctness validation
> 
> It's so strange. I'll go check if some tables of the generated tpcds data are empty.

Yes the tables are valid. It seems 200M is still too small, i'll check if 500M or more works.